src/components/List/ListType1/ListType1.jsx

```diff
@@ -0,0 +1,34 @@
+import "./ListType1.style.scss";
+import defaultIcon from "../../../assets/images/mainCate__itemIcon01.png";
+import defaultThumb from "../../../assets/images/mainCate__itemBg01.jpg";
+
+const ListType1 = ({ items }) => {
+  return (
+    <div className="listType1">
+      <div
+        className="listType1__list"
+        data-scroll
+        data-scroll-speed="8"
+        data-scroll-direction="horizontal"
+      >
+        <ul className="list">
+          {items.map((item) => (
+            <li key={item.id} className="list__item">
+              {item.title && (
+                <strong className="list__tit">{item.title}</strong>
+              )}
```

Comment:
아직 타입스크립트를 쓰지 않아 title로 어떤 값이 들어올지를 모르기 때문에 삼항 연산자로 표현하는 걸 추천합니다. 
```suggestion
              {item.title ? (
                <strong className="list__tit">{item.title}</strong>
              ) : null}
```

--------
예를 들어 아래와 같은 상황에서는 0이 ui에 노출됩니다.
0 && (...)